src/js/modules/openMovieModal.js

```diff
@@ -1 +1,16 @@
-import { fetchItems } from './fetchItems';
+// import { fetchItems } from './fetchItems';
+
+(() => {
+  const refs = {
```

Comment:
to tworzy glob zmienna (BAD).
Przepisz kod (samo-wyw sie funkcje) na ES6 MODULE (import/export).